packages/wallets/src/features/cashier/modules/WithdrawalFiat/__tests__/WithdrawalFiat.spec.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { useCashierFiatAddress } from '@deriv/api';
+import { act, render, screen, waitFor } from '@testing-library/react';
+import WithdrawalFiat from '../WithdrawalFiat';
+
+jest.mock('@deriv/api', () => ({
+    useCashierFiatAddress: jest.fn(),
+}));
+
+describe('<WithdrawalFiat />', () => {
```

Comment:
within the code of component, I see this hook:

![Screenshot 2023-12-19 at 09 36 51](https://github.com/binary-com/deriv-app/assets/141034155/89bf0bd0-b853-41b2-a8a9-4cce73e52eef)

what does it actually do / should it be tested etc?
it _feels_ like important part of component, but I haven't managed to quickly figure out exact logic,

Reply:
whats even the use of verificationCode property in that component? does that mean that whenever verification code change, we submit withdrawal? sounds risky (I know it was already there before this PR - but side effects of writing tests is to discover edge cases / incorrect code etc)

Reply:
@wojciech-deriv, oh 😓, will include this part in the test cases.
Since, this component handles withdrawal for fiat, we make use of the verificationCode to request for a third-party DoughFlow iframe url from our BE. 
I am not sure why is this added as a dependency to the useEffect 😕.
Maybe @lubega-deriv can help.

Reply:
@wojciech-deriv @aum-deriv I believe that useEffect was added in the previous implementation where we fetched the verification code from sessionStorage, hence why we had to fetch and update the verificationCode sent to the useMutation hook. But since we updated the implementation and the verification code is being fetched from the url instead we could probably remove that section 🤔

Reply:
Noted @lubega-deriv 👍. Maybe, we can have a separate card for it 🤔.

Reply:
thanks, sure, as long as there is a tech debt card somewhere added (feels pretty important to get this bit right)
@aum-deriv @lubega-deriv